rust-code-analysis-cli/src/web/server.rs

```diff
@@ -633,7 +633,7 @@ mod tests {
                        "end_line": 4,
                        "metrics": {"cyclomatic": {"sum": 2.0, "average": 1.0},
                                    "cognitive": {"sum": 0.0, "average": 0.0},
-                                   "nargs": {"sum": 0.0, "average": 0.0},
+                                   "nargs": {"functions": 0.0, "closures": 0.0, "total": 0.0, "average": 0.0},
```

Comment:
Maybe we should have total, average, total_functions, average_functions, total_closures, average_closures

Reply:
I can add them